src/Symfony/Component/Uid/CHANGELOG.md

```diff
@@ -1,6 +1,5 @@
 CHANGELOG
 =========
-
```

Comment:
Please add back this line

Reply:
i have added it

Reply:
It is still present and should be reverted, like the other line in this file